script.module.resolveurl/lib/resolveurl/plugins/reviewtech.py

```diff
@@ -23,7 +23,7 @@
 class ReviewTechResolver(ResolveGeneric):
     name = 'ReviewTech'
     domains = ['reviewtech.me']
-    pattern = r'//(.*?\.reviewtech\.me)/(?:embed-)?([0-9a-zA-Z]+)'
+    pattern = r'//(.*?\.reviewtech\.me)/(?:embed-embed-)?([0-9a-zA-Z]+)'
```

Comment:
This wont match embed urls of the format `http://reviewtech.me/embed-bxgs722nsd.html`
Is that intentional?

Reply:
Yes